ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -116,6 +120,17 @@ trait ParameterStorageBackend {
 
   /** Returns the ledger identity parameters, or None if the database hasn't been initialized yet. */
   def ledgerIdentity(connection: Connection): Option[ParameterStorageBackend.IdentityParams]
+
+  /** Initialize the ledger metering end parameters if unset */
+  def initializeLedgerMeteringEnd(init: LedgerMeteringEnd)(connection: Connection)(implicit
```

Comment:
can you please move these to it's own trait / and the implementation to it's own file?
that would keep this parameters table related independent from ledger metering

Reply:
(I would not mind having one metering related trait, if you have concerns having it too granular)